### PR TITLE
VSBuildV1 - added ability to disable default logger

### PR DESCRIPTION
**Pull request by anatolybolshakov**
**Task name**: VSBuildV1

**Description**: 
- Added input 'enableDefaultLogger' - which allows to disable default logger (it is 'true' by default for backward compatibility).
- Added 'IsDefaultLoggerEnabled' parameter for related methods - to control default logger usage - updated MSBuildHelpers common package

**Documentation changes required:** Y

**Added unit tests:** added tests to cover new input behavior. Amended current tests to consider new parameter.

**Attached related issue:** https://github.com/microsoft/azure-pipelines-tasks/issues/14255

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected:
 Tested changes:
 - with 'enableDefaultLogger' disabled - logs with msbuild args:

image

 - with 'enableDefaultLogger' enabled: - logs with msbuild args:

image

### CODE REVIEW

File: Tasks/Common/MSBuildHelpers/InvokeFunctions.ps1 [hunk 1]
```diff
@@ -30,15 +31,15 @@ function Invoke-BuildTools {
...
+                Invoke-MSBuild -ProjectFile $file -Targets Clean -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger -IsDefaultLoggerEnabled:$IsDefaultLoggerEnabled @splat
             }
 
             # If we cleaned and passed /t targets, we don't need to run them again
             if (!$Clean -or $MSBuildArguments -notmatch "[/-]t(arget)?:\S+") {
                 if ($CreateLogFile) {
                     $splat["LogFile"] = "$file.log"
                 }
-                Invoke-MSBuild -ProjectFile $file -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger @splat
+                Invoke-MSBuild -ProjectFile $file -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger -IsDefaultLoggerEnabled:$IsDefaultLoggerEnabled @splat
```
Comment (DaniilShmelev) [inline Tasks/Common/MSBuildHelpers/InvokeFunctions.ps1:42]:
```suggestion
                Invoke-MSBuild -ProjectFile $file -MSBuildPath $MSBuildLocation -AdditionalArguments $MSBuildArguments -NoTimelineLogger:$NoTimelineLogger -IsDefaultLoggerEnabled:$IsDefaultLoggerEnabled @splat
```

**Review by DaniilShmelev — APPROVED** [review]
LGTM

File: Tasks/Common/MSBuildHelpers/InvokeFunctions.ps1 (hunk 1 above)
Comment (anatolybolshakov) [reply Tasks/Common/MSBuildHelpers/InvokeFunctions.ps1]: Thanks! Fixed it

**Review by alexander-smolyakov — APPROVED** [review]

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**